src/components/Employees/index.tsx

```diff
@@ -0,0 +1,214 @@
+                        if (!employeeExists(employee, filteredEmployees)) {
+                            filteredEmployees.push(employee);
+                        }
+                    }
+                });
+            });
+        }
+        return filteredEmployees;
+    }
+
+    return (
+        <div className={styles['employees']}>
+            <h1 className={styles['page-title']}>
+                {t('description.employees')}
+            </h1>
+            <div className={styles['input-search-icon-wrapper']}>
+                <input
+                    className={styles['input']}
+                    placeholder={t('description.search')}
+                    onChange={(event) => {
+                        const search = event.target.value;
+                        if (search) {
+                            searchParams.set('search', search);
+                            setSearchParams(searchParams);
+                        } else {
+                            searchParams.delete('search');
+                            setSearchParams(searchParams);
+                        }
+                    }}
+                />
+                <FontAwesomeIcon icon={faSearch} className={styles['icon']} />
+            </div>
+            <div className={styles['search-labels-wrapper']}>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('Sensei', 'filter[]')}
+                >
+                    Sensei
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('Front-End', 'filter[]')}
+                >
+                    Front-End
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('Back-End', 'filter[]')}
+                >
+                    Back-End
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('QA', 'filter[]')}
+                >
+                    QA
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('DevOps', 'filter[]')}
+                >
+                    DevOps
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() => addFilter('Full-Stack', 'filter[]')}
+                >
+                    Full-Stack
+                </button>
+                <button
+                    className={styles['search-button']}
+                    onClick={() =>
+                        addFilter(t('description.available'), 'filter[]')
+                    }
+                >
+                    {t('description.available')}
+                </button>
```

Comment:
This should be rendered as a list, by extracting filters to a constant and then iterating over that array
example: 
```
const filters = ['Sensei', 'Front-end']

filters.map(item => <button key={item} onClick={() => addFilter(item, 'filter[]')} className={styles['search-button']}>{item}</button>

Reply:
Good idea!